lib/static/static/js/controllers/repositoriesCtrl.js

```diff
@@ -1,29 +1,67 @@
 (function() {
-  angular.module('huburn').controller('repositoriesCtrl', ['$scope', '$http', 'navigationService', function($scope, $http, navigationService) {
-    $scope.repositories = [];
+  angular
+    .module('huburn')
+    .controller('repositoriesCtrl', repositoriesCtrl);
+
+  repositoriesCtrl.$inject = ['$scope', '$http', 'navigationService'];
 
+  function repositoriesCtrl($scope, $http, navigationService) {
+    
+    $scope.repositories = [];
     navigationService.clear();
 
-    $http.get('/github', { params: { path: '/user/repos' } }).success(function(repos) {
-      addRepos(repos);
-    });
+    var addUserRepos = function() {
+      $http.get('/github', { params: { path: '/user/repos' } }).success(function(repos) {
```

Comment:
There is a githubService you can use that does the $http parsing.  See the burnupCtrl for examples on how to use it.